app/controllers/avo/base_controller.rb

```diff
@@ -504,8 +504,10 @@ def destroy_success_action
     end
 
     def destroy_fail_action
+      flash[:error] = destroy_fail_message
+
       respond_to do |format|
-        format.html { redirect_back fallback_location: params[:referrer] || resources_path(resource: @resource, turbo_frame: params[:turbo_frame], view_type: params[:view_type]), error: destroy_fail_message }
+        format.turbo_stream { render partial: "avo/partials/flash_alerts" }
```

Comment:
Are you sure that removing the `redirect` does not break other flows?

Reply:
None that I tested or that is covered by automatic tests